demo/src/app/components/+modal/demos/auto-shown/auto-shown.html

```diff
@@ -1,9 +1,10 @@
 <button type="button" class="btn btn-primary" (click)="showModal()">Render auto-shown modal</button>
-<div *ngIf="isModalShown" [config]="{ show: true }" (onHidden)="onHidden()" bsModal #autoShownModal="bs-modal" class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+<div *ngIf="isModalShown" [config]="{ show: true }" (onHidden)="onHidden()" bsModal #autoShownModal="bs-modal" class="modal fade"
+     tabindex="-1" role="dialog" aria-hidden="true" aria-labelledby="dialog-auto-name" aria-modal="true">
```

Comment:
aria-hidden attribute has to be removed or changed to `false` when the modal is opened

Reply:
this is for all the demos you've changed in this PR